### PR TITLE
modified for new version of selenium

### PR DESCRIPTION
**Pull request by kazem3d**

### CODE REVIEW

File: scraper.py [hunk 1]
```diff
@@ -1,46 +1,73 @@
 from selenium import webdriver
 import time
 import sys
+from selenium.webdriver.common.by import By
+
+driver = webdriver.Firefox()
```
Comment (AgiMaulana) [inline scraper.py:6]: do we need to update the `Readme.md`?

**Comment by kazem3d** [comment]
Please let me check?

File: scraper.py (hunk 1 above)
Comment (kazem3d) [reply scraper.py:6]: if we replace Firefox driver with Chrome , then we dont need to change Read.md

**Review by AgiMaulana — APPROVED** [review]

File: scraper.py (hunk 1 above)
Comment (AgiMaulana) [reply scraper.py:6]: If using Firefox doesn't require an additional web driver, then it's good. I've approved, you can merge by yourself now. Thank you

Comment (kazem3d) [reply scraper.py:6]: thanks for your approval but why the code dosent merge?

Comment (AgiMaulana) [reply scraper.py:6]: Merged.

I thought the requester could merge the PR by themselves once approved.

**Pull request merged**